tests/unittests/CoreGraphics.drawing/CTDrawingTests.cpp

```diff
@@ -26,7 +26,7 @@ static NSURL* __GetURLFromPathRelativeToModuleDirectory(NSString* relativePath)
 }
 #endif // WINOBJC
 
-DISABLED_DRAW_TEST_F(CTFrame, BasicDrawingTest, WhiteBackgroundTest) {
+TEXT_DRAW_TEST_F(CTFrame, BasicDrawingTest, WhiteBackgroundTest<PixelByPixelImageComparator<ComparisonMode::Mask>>) {
```

Comment:
this is a bad example since it's not the default case, but you know what i meant. :P

---
In reply to: [95863602](https://github.com/Microsoft/WinObjC/pull/1692#discussion_r95863602) [](ancestors = 95863602)